modules/cudf/src/series/bool.ts

```diff
@@ -28,4 +30,97 @@ export class Bool8Series extends NumericSeries<Bool8> {
   get data() {
     return new Uint8ClampedBuffer(this._col.data).subarray(this.offset, this.offset + this.length);
   }
+
+  _prepare_scan_series(skipna: boolean) {
+    if (skipna) { return this; }
+
+    // TODO: skipna=false
+    return this;
+  }
+
+  /**
+   * Compute the cumulative max of all values in this Series.
+   *
+   * @param skipna The optional skipna if true drops NA and null values before computing reduction,
+   * else if skipna is false, reduction is computed directly.
```

Comment:
Our current implementations of anything related to `na` only handles `na` values.

For example, `dropNaNs` only dropping `NA` values, and `dropNulls` only handling nulls. So whenever `skipna` is handled, I think it would make sense to only remove `NA` values, and update this doc string.

Reply:
I actually just copied this from `mean`:

https://github.com/rapidsai/node-rapids/blob/abdab8772eb0fa5f8455516f48acf714bfe22088/modules/cudf/src/series/float.ts#L167-L168

But I do think it probably could be worded better. E.g. in `float.ts` it could be "NA and NaN values", and for bool and int, it could just be "NA values". That said we do convert nulls to NA in the constructors so maybe it's good to include "null" to reflect that. Do you have thoughts or preferences @trxcllnt ?

Reply:
Not sure what the question is? We only have the concept of nulls (represented in the `nullMask`), and `NaN` (which are valid floating point values in the `data` buffer).

Pandas terminology is confusing here as they represent "missing" values as NaN sentinels in numeric Series (but then have no way to differentiate between true NaNs and nulls, i.e. Python's `None`), and `None` in non-numeric Series (i.e. strings).

Reply:
@trxcllnt would `skipNulls` be a better param name for node-rapids then? (throughout)

Reply:
That seems reasonable to me.

Reply:
@bryevdv yeah I definitely think that would reduce confusion

Reply:
:+1: since it will affect files otherwise outside the scope of this PR, I'll open a new issue.